### PR TITLE
Fix overwrap double quote when it has double quote inside - Closes #25

### LINKED ISSUE #25: String argument with double quotation does not work
### Description
When double quotation is used within an argument, it will convert with double quotation again.
As a result, it will cut the argument in a wrong position.

```
vorpal command '{"test": 1}'
```
this will result in an argument as
```
{
```

### PR DESCRIPTION
**Pull request by shuse2**
Closes #25

### CODE REVIEW

File: lib/vorpal.js [hunk 1]
```diff
@@ -169,7 +169,8 @@ Vorpal.prototype.parse = function (argv, options) {
           continue;
         }
         if (args[i].indexOf(' ') > -1) {
-          args[i] = `"${args[i]}"`;
+          var quote = args[i].indexOf('"') > -1 ? '\'' : '"';
+          args[i] = `${quote}args[i]${quote}`;
```
Comment (willclarktech) [inline lib/vorpal.js]: `${quote}${args[i]}${quote}`!

File: dist/vorpal.js [hunk 2]
```diff
@@ -169,7 +169,8 @@ Vorpal.prototype.parse = function (argv, options) {
           continue;
         }
         if (args[i].indexOf(' ') > -1) {
-          args[i] = '"' + args[i] + '"';
+          var quote = args[i].indexOf('"') > -1 ? '\'' : '"';
+          args[i] = quote + 'args[i]' + quote;
```
Comment (willclarktech) [inline dist/vorpal.js]: Why the quotes around `args[i]`?

File: test/vorpal.js [hunk 3]
```diff
@@ -108,6 +108,16 @@ describe('argument parsing', function () {
...
 
+  it('should execute a command with a required arg which has double quote', function () {
+    var fixture = obj({ options: {}, str: '{"test": 1}' });
+    obj(vorpal.execSync('required \'{"test": 1}\'')).should.equal(fixture);
+  });
+
+  it('should execute a command with a required arg which has single quote', function () {
+    var fixture = obj({ options: {}, str: "{'test': 1}" });
+    obj(vorpal.execSync('required "{\'test\': 1}"')).should.equal(fixture);
+  });
```
Comment (willclarktech) [inline test/vorpal.js:119]: Tests are passing somehow?

**Review by willclarktech — CHANGES_REQUESTED** [review]
Should we merge into a feature branch (patch release)?

File: test/vorpal.js (hunk 3 above)
Comment (shuse2) [reply test/vorpal.js:119]: it seems like these tests are not very useful =(

**Review by willclarktech — APPROVED** [review]

**Pull request merged**